installation/proc-install-rhdh-operator-admin.adoc

```diff
@@ -0,0 +1,53 @@
+[id='proc-install-rhdh-operator_{context}']
+= Installing {product} using the Operator
+
+[IMPORTANT]
+====
+The {product} Operator is a Technology Preview feature only.
+
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+====
+
+This section describes the installation procedure for installing {product} on your OpenShift Container Platform using the Operator.
+
+//You can use the {product} Operator in Red Hat OpenShift (OpenShift) to install {product}.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift web console.
+* You have configured the appropriate roles and permissions within your project to create an application. See the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications] for more details.
+* If installing a CI build from quay.io/rhdh, make sure that:
```

Comment:
Remove this entire block -- we should not document in official guides how to install pre-GA bits.